mocha/mocha.opts

```diff
@@ -0,0 +1,4 @@
+--recursive
+--webpack-config webpack.config.js
+--require mocha/setup.js
+--timeout 10s
```

Comment:
10s is a lot, I agree. But why shouldn't the tests have any reason to take more than 2s? Setting up a dom with interaction seems an heavy operation. Espacially with JSDom which is JS. Plus, VueJS is not sync. It has an render loop which caps the render frequency just like the render loop does in a video game. This is why I need to call `Vue.prototype.$nextTick` in some tests. Travis may be slow.

The guilty build is [this one](https://travis-ci.org/jaywink/socialhome/builds/267342961).
[This test, for instance](https://travis-ci.org/jaywink/socialhome/jobs/267342963#L1449) took 1.9 secs. I think 4 secs is a fairly trade off. What do you think?